### PR TITLE
Added in GenericServlet and HttpServlet Snippets

### LINKED ISSUE #9: Jakarta Servlet GenericServlet and HttpServlet snippets
Add the following Jakarta Servlet snippets for Java files:

- extending GenericServlet
- extending HttpServlet

The snippet should provide the corresponding imports and methods that are used in best practice.

https://jakarta.ee/specifications/servlet/5.0/apidocs/jakarta/servlet/GenericServlet.html
https://jakarta.ee/specifications/servlet/5.0/apidocs/jakarta/servlet/http/HttpServlet.html

See https://eclipse-ee4j.github.io/jakartaee-tutorial/servlets001.html#BNAFE and https://github.com/eclipse-ee4j/jakartaee-tutorial-examples/tree/master/web/servlet for examples.

*note the tutorial is referring to Jakarta EE 8, with Jakarta EE 9 the package names will change to jakarta... which is why they are slightly different in the apidocs.

### PR DESCRIPTION
**Pull request by AnkushSharma2698**

### CODE REVIEW

File: jakarta.ls/src/main/resources/servlet.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            "@WebServlet(name = \"${2:hello}\", urlPatterns = { \"${3:/hello}\" })",
+            "public class ${4: classname} extends HttpServlet {",
+            "\t@Override",
+            "\tprotected void doGet(HttpServletRequest req, HttpServletResponse res) throws ServletException, IOException {",
+            "\t\tres.setContentType(\"text/html;charset=UTF-8\");",
+            "\t\tres.getWriter().println(\"Hello Jakarta EE 9!\");",
+            "\t}",
+            "}"
+        ],
+        "description": "HttpServlet - HTTP servlet suitable for a Web site",
```
Comment (kathrynkodama) [inline jakarta.ls/src/main/resources/servlet.json:47]: The description should explain that which method it is overriding (doGet), for example: "HttpServlet - supports HTTP GET requests"

Also considering adding a second HTTPServlet request snippet that overrides the doPost method, will require research on best practice implementations of doPost (but should be very similar to this snippet)

File: jakarta.ls/src/main/resources/servlet.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+            "import jakarta.servlet.http.HttpServletRequest;",
+            "import jakarta.servlet.http.HttpServletResponse;",
+            "import java.io.IOException;",
+            "",
+            "@WebServlet(name = \"${2:hello}\", urlPatterns = { \"${3:/hello}\" })",
+            "public class ${4: classname} extends HttpServlet {",
+            "\t@Override",
+            "\tprotected void doGet(HttpServletRequest req, HttpServletResponse res) throws ServletException, IOException {",
+            "\t\tres.setContentType(\"text/html;charset=UTF-8\");",
+            "\t\tres.getWriter().println(\"Hello Jakarta EE 9!\");",
```
Comment (kathrynkodama) [inline jakarta.ls/src/main/resources/servlet.json:43]: Lets change the message here to match the GenericServlet snippet, something like "Example HTTP GET request"

File: jakarta.ls/src/main/resources/servlet.json [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+            "package ${1: packagename}",
+            "",
+            "import jakarta.servlet.ServletException;",
+            "import jakarta.servlet.annotation.WebServlet;",
+            "import jakarta.servlet.http.HttpServlet;",
+            "import jakarta.servlet.http.HttpServletRequest;",
+            "import jakarta.servlet.http.HttpServletResponse;",
+            "import java.io.IOException;",
+            "",
+            "@WebServlet(name = \"${2:hello}\", urlPatterns = { \"${3:/hello}\" })",
```
[the remark this answers is not in the document]
Comment (AnkushSharma2698) [reply jakarta.ls/src/main/resources/servlet.json:38]: I will remove it to keep the functionality of the snippet as simple as possible

**Comment by kathrynkodama** [comment]
Can you resolve conflicts and add the DCO check

**Pull request closed without merging**